website/source/docs/builders/triton.html.md

```diff
@@ -130,17 +131,20 @@ cloud:
     {
       "type": "triton",
       "triton_account": "triton_username",
-      "triton_key_id": "6b:95:03:3d:d3:6e:52:69:01:96:1a:46:4a:8d:c1:7e",
-      "triton_key_material": "~/.ssh/id_rsa",
-      "source_machine_name": "image-builder",
-      "source_machine_package": "g3-standard-0.5-smartos",
+      "triton_key_id": "90:30:b4:f8:c7:d6:7b:b9:f7:e1:8b:43:bd:00:00:00",
+      "triton_key_material": "/Users/rickard/.ssh/joyent_id_rsa",
+      "source_machine_package": "g4-highcpu-512M",
       "source_machine_image": "70e3ae72-96b6-11e6-9056-9737fd4d0764",
+      "source_machine_networks": [
+        "1fc62c97-c1f0-41c6-9ef7-1f7ebe0ff09a",
+        "f7ed95d3-faaf-43ef-9346-15644403b963"
+      ],
       "ssh_username": "root",
-      "ssh_private_key_file": "~/.ssh/id_rsa",
+      "ssh_private_key_file": "/Users/rickard/.ssh/joyent_id_rsa",
```

Comment:
(Unless someone recently added tilde expansion and I didn't notice) tilde expansion is not supported in packer.